<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfDialog</class>
 <widget class="QDialog" name="ConfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxloopOwnMsgs">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Loopback own send Msgs</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name of the Network Device</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRecErrorFrs">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Recive Error Frames</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSilent">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Silent Mode</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNetDev">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCostumDevName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Other</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CostumDevNamelineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>can0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
